app/serializers/atmosphere/action_log_serializer.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Action log serializer.
+#
+module Atmosphere
+  class ActionLogSerializer < ActiveModel::Serializer
+
+    attributes :id, :log_level, :message, :created_at
```

Comment:
Ok my bad. That being said the answer to your question is different :). action_id is not needed as the action log is embedded in action serializer.